app/helpers/mountain_project.py

```diff
@@ -1,14 +1,17 @@
-from ..errors.exeptions import *
 import io
+import os
+from typing import Dict, Optional, Union
+
 import pandas as pd
-from pandas.errors import EmptyDataError, ParserError
 import requests
 from requests import ConnectionError, ConnectTimeout, HTTPError, ReadTimeout, Timeout
-from typing import Dict, Optional, Union
+from pandas.errors import EmptyDataError, ParserError
+
+from ..errors.exeptions import *
 
 
 _DEV_USER_DATA = {"status": 0, "name": "Dev", "mp_id": 1111}
-_DEV_TEST_TICKS = "test_ticks.csv"
```

Comment:
This change breaks the `TestMountainProjectHandler.test_mp_dev_env_parse_ticks` test when I run `pytest` in the root directory (as I have in the past). Passes when I run `pytest` from the `app` directory (as expected). I'll update the *Testing* section  of the README to reflect that `pytest` needs to be run under `app` now.

Reply:
@zachtheclimber, Hmm that's strange, I have been running the tests from the root project directory, I only made this change because for some reason the relative file path was no longer working for me which is why I added `os.path.join(os.getcwd(), 'test_ticks.csv')` to point to the test_ticks file. Could be an issue on my end if it was working for you in the past. Honestly the only reason I went down this path was because that test was broken when I tried to troubleshoot the error handling tests. Either way feel free to put this back the way it was :)

Reply:
That is strange. Yeah, I've been running tests from the root dir all along. I think the change is good. More explicit and we don't really need the test_ticks file out in the root dir anyway. I don't see a reason to revert, especially if it was breaking the test on your end.